packages/extension-ui/src/Popup/AuthManagement/DisconnectApp.tsx

```diff
@@ -35,15 +35,15 @@ function DisconnectApp({ className }: Props): React.ReactElement<Props> {
   const onAction = useContext(ActionContext);
   const { show } = useToast();
 
-  const goToAuthList = useCallback(() => onAction('/auth-list'), [onAction]);
+  const goTo = useCallback((path: string) => () => onAction(path), [onAction]);
```

Comment:
Maybe we can create a helper with this fn to import when needed? Seems a lot of places has the same code :)